tools/spritemaker.py

```diff
@@ -875,6 +875,15 @@ def run_gss(sprite_db, sprite_db_keys, quiet=False, sp_per_file=500):
         if not menutils.ask_continue():
             return
 
+    #Refresh this so we make all invalid sprites
```

Comment:
what is this for?

Reply:
Running the spritemaker and just hitting `Generate Sprites` (in the case you just added an `image monika <code>` in the file won't generate it's corresponding ATL required staticsprite. The part there allows it to do so (needed to especially here so the tears could be updated)